internal/binutils/binutils.go

```diff
@@ -179,6 +179,27 @@ func findObjdump(paths []string) (string, bool, bool) {
 	return "", false, false
 }
 
+// findNm finds and returns path to preferred nm binary.
+// Order of preference is: llvm-nm, nm.
+// On MacOS only, also looks for gnm with least preference.
+// Accepts a list of paths and returns:
+// a string with path to the preferred objdump binary if found,
+// or an empty string if not found;
+// a boolean if any acceptable nm was found.
+func findNm(paths []string) (string, bool) {
+	nmNames := []string{"llvm-nm", "nm"}
```

Comment:
I'd expect somewhere a comment (not here, I'm just commenting here because it's convenient - probably where nm is used) about our confidence that binutils' nm and llvm's nm are interchangeable as it's not obvious at all and depends on what flags are used I guess.

Reply:
Added in where we use `findNm()`.